### ISSUE TITLE
Test battery assumes `singularity` to be installed

### ISSUE THREAD

**Issue by mih** [datalad/datalad-container#233]
This prevents meaningful testing on windows (there is no singularity for that platform). Tests need to be made conditional re availability of singularity.

**Comment by yarikoptic** [comment]
Well worth making them conditional indeed. Related: Is docker installed on windows CI?  I wonder if it is worth  adopting https://github.com/ReproNim/containers/blob/master/scripts/singularity_cmd#L139 -- that one is a bash script and works on OSX to invoke singularity through docker.

**Comment by adswa** [comment]
> Is docker installed on windows CI? 

Yes, in the latest three available Windows Appveyor images. See https://www.appveyor.com/docs/windows-images-software/
